apps/cowswap-frontend/src/common/updaters/orders/UnfillableOrdersUpdater.ts

```diff
@@ -44,9 +44,7 @@ export function UnfillableOrdersUpdater(): null {
   const updatePendingOrderPrices = useSetAtom(updatePendingOrderPricesAtom)
   const isWindowVisible = useIsWindowVisible()
 
-  const pendingLimit = useOnlyPendingOrders(chainId, UiOrderType.LIMIT)
-  const pendingTwap = useOnlyPendingOrders(chainId, UiOrderType.TWAP)
-  const pending = useMemo(() => pendingLimit.concat(pendingTwap), [pendingLimit, pendingTwap])
+  const pending = useOnlyPendingOrders(chainId)
```

Comment:
We stopped checking for SWAPs since [the preparation for TWAP](https://github.com/cowprotocol/cowswap/issues/3561#issuecomment-1878914120).
This change checks for all order types once again, since we need the indication for all pending orders.